src/plugins/ceramic-feed/server/services/ceramic/CeramicClient.js

```diff
@@ -26,38 +31,94 @@ class CeramicClient {
       (fields, [field, relatedSchema]) => {
         const relatedMedia = _getMediaFields(relatedSchema)
 
-        return [...fields, relatedMedia.map(
-          relatedField => relatedFieldName(field, relatedField)
-        )]
+        return [
+          ...fields,
+          relatedMedia.map(relatedField =>
+            relatedFieldName(field, relatedField)
+          )
+        ]
       },
       _getMediaFields(schema)
     )
 
     http.interceptors.response.use(({ data }) => Buffer.from(data, 'binary'))
+
+    this.orbisReady = this.initOrbis(strapi)
+
     assign(this, { ipfs, http, mediaFields })
   }
 
+  async initOrbis(strapi) {
+    const { config } = strapi
+    const { ceramicDIDSeed, orbisContext } = config.get('plugin.ceramic-feed')
+    const { Orbis } = await import('@orbisclub/orbis-sdk')
+
+    const sdk = new Orbis()
```

Comment:
need to reformat code a bit - split code blocks with empty lines etc

it's very hard to read